app/src/main/res/values-sk/strings.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0"?>
+<resources>
+    <string name="app_name">eRouška</string>
+    <string name="help_toolbar_title">Nápoveda</string>
+    <string name="go_back_button">Späť do aplikácie</string>
+    <string name="enable_bluetooth_button">Zapnúť Bluetooth</string>
+    <string name="enable">Zapnúť</string>
+    <string name="bt_disabled_title">Zapnite Bluetooth</string>
+    <string name="bt_disabled_desc">Bez zapnutého Bluetooth nemôže eRouška vytvárať zoznam eRoušok vo vašom okolí.\n\nZapnite ho pomocou tlačidla \"Zapnúť Bluetooth\".</string>
+    <string name="dashboard_title_running">eRouška je aktívna</string>
+    <string name="dashboard_title_paused">eRouška je pozastavená</string>
+    <string name="dashboard_body">Aplik\u00e1cia pracuje na pozad\u00ed a monitoruje okolie, pros\u00edm, neukon\u010dujte ju. Nechajte zapnut\u00fd Bluetooth a s telef\u00f3nom pracujte ako obvykle.</string>
```

Comment:
tu je schvalne ten unicode? nie je to konzistentne vo vsetkych stringoch

Reply:
Nějaký magic OneSky. Důležité je, že to v appce funguje správně.